File: Python/1134.py
alcool = 0
diesel = 0
gasolina = 0
combustivel = int(input())
while combustivel != 4:
    combustivel = int(input())
    if combustivel == 1:
        alcool += 1
    if combustivel == 2:
        gasolina += 1
    if combustivel  == 3:
        diesel += 1

print('MUITO OBRIGADO') 
print('Alcool: %d'%alcool)
print('Gasolina: %0d'%gasolina)
print('Diesel: %0d'%diesel)   

File: JavaScript/1134.js
let input = require ('fs').readFileSync('./dev/stdin/file.txt','utf8')
let = lines = input.split('\n')
let combustivel  
alcool = 0
diesel = 0
gasolina = 0
while (combustivel != 4) {
    combustivel =+lines.shift()  
    if (combustivel === 1){
        alcool = alcool + 1
    }else if (combustivel === 2){
        gasolina = gasolina + 1
    }else if (combustivel ===3){
        diesel = diesel +1
    } 
}
console.log('MUITO OBRIGADO')
console.log(`Alcool: ${alcool}`)
console.log(`Gasolina: ${gasolina}`)
console.log(`Diesel: ${diesel}`)

